Documentation/data-collection.md

```diff
@@ -991,6 +991,30 @@ data:
     #
     # apiserver_list_watch_request_success_total:rate:sum represents the rate of change for successful LIST and WATCH requests over a 5 minute period.
     - '{__name__="apiserver_list_watch_request_success_total:rate:sum", verb=~"LIST|WATCH"}'
+    #
+    # owners: (https://github.com/stackrox/stackrox, @stackrox/eng)
+    #
+    # rhacs:telemetry:rox_central_secured_clusters provides the number of clusters secured by a Central instance of a
+    # Red Hat Advanced Cluster Security installation.
```

Comment:
For future readers, I'd expand a bit on the labels of the metric.

IIUC
* `build`, "release" or "internal".
* `central_id` , unique ID identifying the central cluster.
* `central_version`, the product's full version.
* `hosting`, "cloud-service" or "self-managed".
* `install_method`, "operator", "manifest" or "helm".